opm/material/common/Tabulated1DFunction.hpp

```diff
@@ -35,6 +35,7 @@
 #include <iostream>
 #include <tuple>
 #include <vector>
+#include "fmt/format.h"
```

Comment:
Is this used in tests in this module? Internal libfmt is not available in this repo.

Reply:
sorry, did not know about that. Should we include Internal libfmt here or I can change the code without using `fmt::format`?

Either way will work for me.

Reply:
I think for now it is better to use an sstream than go through the hassle of adding internal libfmt. Hopefully we will merge this repo with opm-common soon, atleast that has is the plan, and then the problem goes away..

Reply:
Sure. I will do that.